### ISSUE TITLE
Server side validation errors

### ISSUE THREAD

**Issue by dmitov** [semisleep/simple-vue-validator#45]
Hi there! 

I reviewed the package and got to say that it's the most clean and nice way to validate forms, I love it!

However, there are some cases when the frontend validation passes, but the backend one will fail. For example, I got many users registering at the same time, and there might be some collisions with the emails, so I check on the backend API (Laravel) once again if the whole submitted form is valid. If not, it returns a normal 422 from Laravel with the error messages bag.

My question here is how to trigger those validation errors to display on the frontend? I'd like to do something like:

```
this.$invalidate([
    { email: 'This email is already taken' }, 
    {username: 'This username is already taken'}
]);
```

Basically a way to trigger custom error messages on the fly. Is that possible at the moment?
